modules/javafx.graphics/src/main/java/com/sun/glass/ui/win/WinAccessible.java

```diff
@@ -299,14 +299,16 @@ public void sendNotification(AccessibleAttribute notification) {
                 break;
             }
             case INDETERMINATE: {
-                if (getAttribute(ROLE) == AccessibleRole.CHECK_BOX) {
+                if (getAttribute(ROLE) == AccessibleRole.CHECK_BOX
+                        || getAttribute(ROLE) == AccessibleRole.CHECK_BOX_TREE_ITEM) {
```

Comment:
I think the proposed change is less intrusive than a switch, but I don't mind either way. It might make sense to use a local variable to avoid calling `getAttribute` twice.